jp/components/calendar.md

```diff
@@ -286,11 +286,12 @@ This is the result.
 </div>
```

Comment:
I'm unable to add a comment on line 285, but could you please localize the button text?